File: app/src/main/java/com/treelzebub/readerbility/util/ReadabilityService.java
package com.treelzebub.readerbility.util;

import com.treelzebub.readerbility.thing.Bookmark;

import java.util.List;

import retrofit.client.Response;
import retrofit.http.GET;
import retrofit.http.Path;

/**
 * Created by Tre Murillo on 2/27/15.
 */
public interface ReadabilityService {


    @GET("GET /api/rest/v1/bookmarks/{bookmark_id}")
    List<Bookmark> getBookmark(@Path("bookmark_id") String bookmark_id);
}

File: app/src/main/java/net/treelzebub/readerbility/auth/ReadabilityService.kt
package net.treelzebub.readerbility.auth

import net.treelzebub.readerbility.api.model.Article
import net.treelzebub.readerbility.api.model.User
import retrofit.http.GET
import retrofit.http.Path

/**
 * Created by Tre Murillo on 1/1/16
 */
interface ReadabilityService {

    @GET("/users/_current")
    fun identity(): User

    @GET("/articles/{article_id}")
    fun articles(@Path("article_id") id: String): List<Article>

    // ...
}
